INN/umbrella-ipbs
# Homepage features widget styles

## Changes

This pull request makes the following changes:

- Styles a Largo Recent Posts widget on the homepage bottom, with "Default" widget background set
- turns `.widgettitle` styles into a mixin, and adds logic to resize the text on mobile
- turns the homepage shadow styles into a mixin
- copies in `partials/widget-content.php` from Largo to add the `morelink` link.
- uses a placeholder style for `.btn`, until #71 is merged.

![Screen Shot 2020-01-29 at 12 59 05](https://user-images.githubusercontent.com/1754187/73383394-3ed71780-4297-11ea-9a95-644db539adb2.png)
![Screen Shot 2020-01-29 at 12 59 30](https://user-images.githubusercontent.com/1754187/73383396-3ed71780-4297-11ea-9e10-ba691c56159f.png)



## Why



Resolves #59

## Testing/Questions

Features that this PR affects:

- widgettitle styles


Questions that need to be answered before merging:

- [x] ~This is waiting for the button styles in #71 to be merged, which will cause a merge conflict in `less/mixins.less`~ resolved in 44604d6aee40c75293a0cc577cbddbfc4b47532e
- [x] ~This still needs the blue swooshy background image to be added; I don't have access to that today.~ resolved in e6e3ce8749da99c2dee8e989591b78189f07fbab
- [x] Why is the shadow cut off? 
    ![Screen Shot 2020-01-29 at 12 57 16](https://user-images.githubusercontent.com/1754187/73383308-13542d00-4297-11ea-8630-845a1f817454.png)
- [x] Why is the button background cut off?
    ![Screen Shot 2020-01-29 at 12 57 13](https://user-images.githubusercontent.com/1754187/73383319-1818e100-4297-11ea-9047-cdd4f74be69c.png)

Steps to test this PR:

0. Make sure that the three most-recent posts in a category have featured images
1. Add a Largo Recent Posts widget to the Homepage Bottom widget area:
    - title: Features Heading
    - 3 posts
    - avoid duplicates
    - large thumbnail
    - custom excerpt or sentences
    - no byline
    - no publish date
    - no top term
    - limit to that category
